carmajava/interfacemgr/src/main/java/gov/dot/fhwa/saxton/carma/interfacemgr/InterfaceMgr.java

```diff
@@ -204,9 +206,9 @@ public void onNewMessage(cav_msgs.RobotEnabled msg) {
                         if (robotEnabled_) {
 
                             //if robotic control is no longer active then warn and alert all nodes to shut down
-                            if (!msg.getRobotActive()) {
-                                log_.warn("SHUTDOWN", "InterfaceMgr.robotListener senses robot is no longer active at the hardware level.");
-                                publishSystemAlert(AlertSeverity.FATAL, "Robotic control has been disengaged.", null);
+                            if (Math.abs(System.currentTimeMillis() - lastRobotStatusMsgTime_) > robotStatusTimeout_ && !isShutdownUnderway()) {
```

Comment:
It does not matter in this case, But it is still better to use ROS time whenever is possible.